frontend/src/components/dialogs/SecretDialogHCloud.vue

```diff
@@ -0,0 +1,135 @@
+<!--
+SPDX-FileCopyrightText: 2021 23Technologies or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <secret-dialog
+    :value=value
+    :data="secretData"
+    :data-valid="valid"
+    :secret="secret"
+    cloud-provider-kind="hcloud"
+    create-title="Add new HCloud Secret"
+    replace-title="Replace HCloud Secret"
+    @input="onInput">
+
+    <template v-slot:secret-slot>
+      <div>
+        <v-text-field
+        color="primary"
+        v-model="hcloudToken"
+        ref="hcloudToken"
+        label="Hetzner Cloud Token"
+        :error-messages="getErrorMessages('hcloudToken')"
+        @input="$v.hcloudToken.$touch()"
+        @blur="$v.hcloudToken.$touch()"
+        ></v-text-field>
+      </div>
+    </template>
+
+    <template v-slot:help-slot>
+      <div>
+        <p>
+          Before you can provision and access a Kubernetes cluster on Hetzner Cloud, you need to add a HCloud Token.
+          The Gardener needs these credentials to provision and operate HCloud infrastructure for your Kubernetes cluster.
```

Comment:
```suggestion
          Before you can provision and access a Kubernetes cluster on Hetzner Cloud, you need to add a Hetzner Cloud Token.
          The Gardener needs these credentials to provision and operate Hetzner Cloud infrastructure for your Kubernetes cluster.
```

Reply:
The Hetzner Cloud documentation always uses the term `Hetzner Cloud`. Only the CLI is called `hcloud`. I think we should also use the full name  `Hetzner Cloud` in the text.